src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -114,6 +114,12 @@ public MainTable(MainTableDataModel model, JabRefFrame frame,
 
         this.pane.getStylesheets().add(MainTable.class.getResource("MainTable.css").toExternalForm());
 
+        //Set sort order column from preferences, currently only single column suported
```

Comment:
As I learned from the SO answer, with shift it's possible to add multiple columns to the sort order.
However, currently only one column is stored, e.g. if you have both author DESCENDING and timestamp ASCENDING, it will only store the first one.  Or do we have an elegant solution to store Key-Value Pairs in the preferences? e.g. a list of columns with sort type.

Reply:
The standard way is to have to lists, one for the keys and one for the values. In the progress, it would be nice if you would add the corresponding wrapper methods (get/putMap) in `JabRefPreferences` for reuse in the future.